spec/amp-cache-guidelines.md

```diff
@@ -56,6 +56,28 @@ Provide guidelines to external parties for implementing their own AMP Caches.
 
 * Participation in [AMP Ads For AMP Pages ("A4A")](https://github.com/ampproject/amphtml/issues/3133) advertisement signing.
 
+## Guidelines: Accepted MIME types
+
+### MIME types for images
+
+Accepted MIME types for images include all `image/` subtypes (e.g. `image/gif`).
+
+### MIME types for fonts
+
+Accepted MIME type prefixes for fonts include the following:
+
+| Media type / subtype  |
+| ------------- |
+|  font/*   (e.g. `font/opentype`)|
```

Comment:
Remove `*`

The others are technically also prefixes.

E.g. `application/font-foo` is supportred.